tests/test-api.c

```diff
@@ -663,6 +663,29 @@ static void testRm(CuTest *tc) {
     CuAssertIntEquals(tc, 5, r);
 }
 
+static void testLoadFile(CuTest *tc) {
+    struct augeas *aug;
+    const char *value;
+    int r;
+
+    aug = aug_init(root, loadpath, AUG_NO_STDINC|AUG_NO_LOAD);
+    CuAssertPtrNotNull(tc, aug);
+    CuAssertIntEquals(tc, AUG_NOERROR, aug_error(aug));
+
+    /* augeas should load a single file */
+    r = aug_load_file(aug, "/etc/fstab");
+    CuAssertRetSuccess(tc, r);
+    r = aug_get(aug, "/files/etc/fstab/1", &value);
+    CuAssertIntEquals(tc, 1, r);
+    CuAssertPtrNotNull(tc, value);
+    CuAssertRetSuccess(tc, r);
```

Comment:
One final thing: we are missing the test that trying to load a nonexistent file does not produce an error; just add this to the end of `testLoadFile`:
```c
    /* augeas should return without an error when trying to load a
       nonexistant file that would be handled by a lens */
    r = aug_load_file(aug, "/etc/mtab");
    CuAssertRetSuccess(tc, r);
    r = aug_match(aug, "/files/etc/mtab", NULL);
    CuAssertIntEquals(tc, 0, r);
```